lib/storage-delegations.js

```diff
@@ -0,0 +1,193 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const assert = require('assert-plus');
+const bedrock = require('bedrock');
+const database = require('bedrock-mongodb');
+const {promisify} = require('util');
+const {util: {BedrockError}} = bedrock;
+
+bedrock.events.on('bedrock-mongodb.ready', async () => {
+  await promisify(database.openCollections)(['zcap-delegation']);
+
+  // FIXME: what are indexes for this collection?
+
+  await promisify(database.createIndexes)([{
+    collection: 'zcap-delegation',
+    fields: {invocationTarget: 1, id: 1},
+    options: {unique: true, background: false}
+  }, {
+    collection: 'zcap-delegation',
+    fields: {controller: 1, id: 1},
+    options: {unique: true, background: false}
+  }]);
+});
+
+/**
+ * Inserts an delegation into storage.
```

Comment:
```suggestion
 * Inserts a delegated zcap into storage.
```

I'm still not sure we should be using "delegation" to refer to a "delegated zcap". People might get confused over what that term means. I realize that we'll be exposing this API as `delegations` ... but we should at least be more descriptive in the docs.